packet.go

```diff
@@ -52,6 +59,10 @@ func Parse(b, secret []byte) (*Packet, error) {
 		return nil, errors.New("radius: invalid packet length")
 	}
 
+	if len(cryptoAuthenticator) != 0 && len(cryptoAuthenticator) != 16 {
```

Comment:
cryptoAuthenticator from the function call is not used anywhere ...?

Reply:
It is used in line 84. (unused values result in an compile-error in go)

Reply:
Sorry... ignore.